### PR TITLE
Disc 657 paging record count

### PR DESCRIPTION
**Pull request by VictorHarbo**
The skeleton for delivering Pagin-Header-Count when indexing had already been made. This PR Updates dependencies to make it work. 

I've locally indexed and the value seems to come through correctly.

### CODE REVIEW

**Review by tokee — CHANGES_REQUESTED** [review]
This adds the core support. What about the "log messages and status"-part of DISC-657?

**Comment by VictorHarbo** [comment]
I've added a couple logs to the `/solr/index`-endpoint as this is the only place we are using continuation streams. 
Regarding status messages, the endpoint already returns a status of eg: `combinedQTime=2860, allDocumentsIndexed=1231516, lastResponseHeader=SolrResponseHeader{rf=1, status=0, qTime=12}`

File: src/main/java/dk/kb/datahandler/util/SolrUtils.java [hunk 1]
```diff
@@ -81,11 +81,13 @@ public static String indexOrigin(String origin, Long sinceTime) throws IOExcepti
         Long documents = 0L;
         String solrResponse = "";
         SolrIndexResponse finalResponse = new SolrIndexResponse();
+        log.info("Starting indexing of records from origin: '{}'", origin);
```
Comment (tokee) [inline src/main/java/dk/kb/datahandler/util/SolrUtils.java:84]: Please add `sinceTime` to the log statement

**Review by tokee — COMMENTED** [review]
Just a tiny extra log-thing, then we're good :pleading_face:

**Pull request merged**